<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="220.80200000000002" tests="8" failures="2">
  <testsuite name="Root Suite" timestamp="2020-02-27T22:19:59" tests="0" failures="0" time="0">
  </testsuite>
  <testsuite name="59c - archive worksheets with multiple sample protocols" timestamp="2020-02-27T22:19:59" tests="8" failures="2" time="220.80200000000002">
    <testcase name="59c - archive worksheets with multiple sample protocols LM-59c-1 - Archive after all samples are complete" time="166.234" classname="LM-59c-1 - Archive after all samples are complete">
    </testcase>
    <testcase name="59c - archive worksheets with multiple sample protocols LM-59c-2 - Archive after failing all samples in the second protocol" time="11.395" classname="LM-59c-2 - Archive after failing all samples in the second protocol">
    </testcase>
    <testcase name="59c - archive worksheets with multiple sample protocols LM-59c-3 - Archive after failing first 3 samples in the second protocol" time="0" classname="LM-59c-3 - Archive after failing first 3 samples in the second protocol">
      <failure message="Timed out retrying: Expected to find element: &apos;.ag-selection-checkbox:not(.ag-hidden)&apos;, but never found it. Queried from element: &lt;div.ag-row.ag-row-no-focus.ag-row-odd.ag-row-level-0.row-failed.ag-row-position-absolute&gt;" type="CypressError"><![CDATA[CypressError: Timed out retrying: Expected to find element: '.ag-selection-checkbox:not(.ag-hidden)', but never found it. Queried from element: <div.ag-row.ag-row-no-focus.ag-row-odd.ag-row-level-0.row-failed.ag-row-position-absolute>
    at Object.cypressErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:65727:11)
    at Object.throwErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:65692:18)
    at Object.throwErrByPath (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:65719:17)
    at retry (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:59237:16)
    at http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:51312:18
    at tryCatcher (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:131273:23)
    at Promise._settlePromiseFromHandler (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:129291:31)
    at Promise._settlePromise (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:129348:18)
    at Promise._settlePromise0 (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:129393:10)
    at Promise._settlePromises (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:129468:18)
    at Async._drainQueue (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:126197:16)
    at Async._drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:126207:10)
    at Async.drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:126081:14)
    at <anonymous>]]></failure>
    </testcase>
    <testcase name="59c - archive worksheets with multiple sample protocols LM-59c-4 - Archive after failing last 3 samples in the second protocol" time="0" classname="LM-59c-4 - Archive after failing last 3 samples in the second protocol">
      <failure message="Timed out retrying: Expected to find element: &apos;.ag-selection-checkbox:not(.ag-hidden)&apos;, but never found it. Queried from element: &lt;div.ag-row.ag-row-no-focus.ag-row-even.ag-row-level-0.row-failed.ag-row-position-absolute&gt;" type="CypressError"><![CDATA[CypressError: Timed out retrying: Expected to find element: '.ag-selection-checkbox:not(.ag-hidden)', but never found it. Queried from element: <div.ag-row.ag-row-no-focus.ag-row-even.ag-row-level-0.row-failed.ag-row-position-absolute>
    at Object.cypressErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:65727:11)
    at Object.throwErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:65692:18)
    at Object.throwErrByPath (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:65719:17)
    at retry (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:59237:16)
    at http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:51312:18
    at tryCatcher (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:131273:23)
    at Promise._settlePromiseFromHandler (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:129291:31)
    at Promise._settlePromise (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:129348:18)
    at Promise._settlePromise0 (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:129393:10)
    at Promise._settlePromises (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:129468:18)
    at Async._drainQueue (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:126197:16)
    at Async._drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:126207:10)
    at Async.drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:126081:14)
    at <anonymous>]]></failure>
    </testcase>
    <testcase name="59c - archive worksheets with multiple sample protocols LM-59c-5 - Archive after failing all samples in the third protocol" time="8.414" classname="LM-59c-5 - Archive after failing all samples in the third protocol">
    </testcase>
    <testcase name="59c - archive worksheets with multiple sample protocols LM-59c-6 - Archive after first sample in the third protocol" time="12.461" classname="LM-59c-6 - Archive after first sample in the third protocol">
    </testcase>
    <testcase name="59c - archive worksheets with multiple sample protocols LM-59c-7 - Archive after last sample in the third protocol" time="12.464" classname="LM-59c-7 - Archive after last sample in the third protocol">
    </testcase>
    <testcase name="59c - archive worksheets with multiple sample protocols LM-59c-8 - Archive after failing all samples in the fourth protocol" time="9.834" classname="LM-59c-8 - Archive after failing all samples in the fourth protocol">
    </testcase>
  </testsuite>
</testsuites>